src/parser/core/modules/CastTime.ts

```diff
@@ -56,15 +57,15 @@ export default class CastTime extends Analyser {
 	 * @returns The index number within the cast time adjustments collection, can be used to reset/end this adjustment later
 	 */
 	public setTimeAdjustment(actions: number[] | 'all', adjustment: number, start: number = this.parser.currentEpochTimestamp, end?: number): number {
-		return this.set(actions, 'time', adjustment, start, end)
+		return this.set(actions, 'time', adjustment, false, start, end)
```

Comment:
this signatures in this file are _very_ quickly approaching, or perhaps have reached, the "this should really be an object parameter" region. not going to block on it - it's probably a seperate PR's worth of changes, but honestly anything over about 3 parameters is in pushing-it territory.

Reply:
Yeah, that was on my mind when I added the recast parameters, but this PR was already ballooning, so I opted to leave that for another day. Would've done the object-param changeover as part of the cast time signature PR if I'd realized I was going to need the recast-oriented options at the time but well... I didn't .-.

TODO note to self to deal with that in the not-too-distant future as we collectively get the cd2/GCD stuff sorted out